.changeset/unlucky-garlics-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: Balance parantheses in error about wrong content type for action
```

Comment:
typo in "parentheses". also, we generally start with a lowercase letter ([see others](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md)).
```suggestion
fix: balance parentheses in error about wrong content type for action
```